cli-plugins/insight/cli-overview.hbs.md

```diff
@@ -4,11 +4,11 @@ The Tanzu Insight CLI plug-in enables querying vulnerability, image, and package
 
 Follow the below steps to install, configure, and use the Tanzu Insight CLI plug-in.
 
->**Note:** Prior to using the CLI plug-in, you must install the Supply Chain Security Tools - Store, either as its own package, or as part of Tanzu Application Platform Build profile or Tanzu Application Platform View profile.
+>**Note:** Prior to using the CLI plug-in, you must install the Supply Chain Security Tools - Store, either as its own package, or as part of Tanzu Application Platform View profile.
 
-1. [Install the Tanzu Insight CLI plug-in](cli-installation.md)
-1. [Configure target endpoint and certificate](../../scst-store/using-encryption-and-connection.md)
-1. [Configure access tokens](../../scst-store/create-service-account-access-token.md)
+1. If the `insight` plug-in is not already installed, see [Install the Tanzu Insight plug-in](cli-installation.md)
+1. [Configure target endpoint and certificate](../../scst-store/using-encryption-and-connection.hbs.md)
```

Comment:
I find it strange that this links back out to the store docs, rather than making use of the cli-plugin section's `cli-configuration` page above which also contains the exact same pieces of information.

Reply:
true, I should link it to that one instead